### ISSUE TITLE
Uncaught TypeError: Cannot read property 'append' of undefined

### ISSUE THREAD

**Issue by sammychiang** [kgrossjo/atom-man#7]
[Enter steps to reproduce below:]

1. ...
2. ...

** Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: [](https://github.com/kgrossjo/-) package, v0.4.0

### Stack Trace

Uncaught TypeError: Cannot read property 'append' of undefined

```
At /Users/jiangzhifeng/./packages//lib/-input-view.coffee:37

TypeError: Cannot read property 'append' of undefined
  at InputView.module.exports.InputView.attach (/Users/jiangzhifeng/./packages//lib/-input-view.coffee:37:31)
  at InputView.module.exports.InputView.initialize (/Users/jiangzhifeng/./packages//lib/-input-view.coffee:14:14)
  at InputView.View (/Users/jiangzhifeng/baiduyun/百度云同步盘/文档/下载/ 2.app/Contents/Resources/app.asar/node_modules/-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at new InputView (/Users/jiangzhifeng/./packages//lib/-input-view.coffee:6:9)
  at -text-editor.module.exports. (/Users/jiangzhifeng/./packages//lib/.coffee:26:29)
  at ComdRegistry.module.exports.ComdRegistry.handleComdEvent (/Users/jiangzhifeng/baiduyun/百度云同步盘/文档/下载/ 2.app/Contents/Resources/app.asar/src/comd-registry.js:241:29)
  at /Users/jiangzhifeng/baiduyun/百度云同步盘/文档/下载/ 2.app/Contents/Resources/app.asar/src/comd-registry.js:3:61
  at Keymapager.module.exports.Keymapager.dispatchComdEvent (/Users/jiangzhifeng/baiduyun/百度云同步盘/文档/下载/ 2.app/Contents/Resources/app.asar/node_modules/-keymap/lib/keymap-ager.js:520:16)
  at Keymapager.module.exports.Keymapager.handleKeyboardEvent (/Users/jiangzhifeng/baiduyun/百度云同步盘/文档/下载/ 2.app/Contents/Resources/app.asar/node_modules/-keymap/lib/keymap-ager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Users/jiangzhifeng/baiduyun/百度云同步盘/文档/下载/ 2.app/Contents/Resources/app.asar/src/window-event-handler.js:180:20)

```

### Comds

```
     -9:16.9.0 find-and-replace:show (-text-editor.editor.is-focused)
     -9:16.6.0 core:paste (-text-editor.editor.mini.is-focused)
  3x -9:15.1.0 core:confirm (-text-editor.editor.mini.is-focused)
     -5:49.9.0 linter:set-bubble-transparent (-text-editor.editor.is-focused)
     -5:49.4.0 lmgtfy:google-it (-text-editor.editor.is-focused)
     -1:23.8.0 settings-view:open (-text-editor.editor.is-focused)
     -1:19.6.0 core:confirm (-text-editor.editor.mini.is-focused)
     -0:01.7.0 : (-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "SFTP-deployment",
      "SFTP-deployment",
      "SFTP-deployment",
      "sourcegraph-"
    ]
  }
}
```

### Installed Packages

```coffee
# User
-beautify, v0.28.7
-copy-on-select, v1.4.0
-django, v0.3.0
-html-preview, v0.1.9
auto-update-packages, v1.0.0
autoclose-html, v0.18.0
autocomplete-python, v0.5.1
color-picker, v2.0.9
highlight-line, v0.11.0
highlight-selected, v0.10.1
html-id-class-snippets, v1.4.1
jsformat, v0.9.0
jshint, v1.3.9
linter, v1.2.0
linter-csslint, v0.0.14
linter-htmlhint, v0.0.17
linter-jshint, v0.1.7
lmgtfy, v0.2.0
, v0.4.0
minimap, v4.10.2
minimap-highlight-selected, v4.3.0
pigments, v0.8.3
remote-edit, v1.8.4
svn, v0.0.6

# Dev
No dev packages
```

**Comment by cornz** [comment]
This should fix your error:

https://github.com/kgrossjo/atom-man/pull/8

**Comment by kgrossjo** [comment]
Yes, indeed it does.  Good work!  Thanks a bunch.  I have merged it.

**Comment by fusion809** [comment]
Um, I have just encountered the same error on Sabayon Linux, with a newly installed Atom 1.0.13.

**Comment by Fishrock123** [comment]
Also just had this error, OS X 10.10, Atom 1.0.19

**Comment by walles** [comment]
@kgrossjo, could you please release a new version with the fix for this applied?

The official 0.4.0 doesn't have the patch and doesn't work.

  Regards /Johan

**Comment by kgrossjo** [comment]
@walles, I apologize for my tardiness.  I should have been more diligent.  So now I have released 0.5.0 with @cornz's fix.  Thanks to both of you for driving this forward.

**Comment by walles** [comment]
Thank you @kgrossjo, it works for me now.
